index.html

```diff
@@ -1566,61 +1551,70 @@ <h4>CoAP Default Vocabulary Terms</h4>
           <tbody>
             <tr>
               <td><code>readproperty</code></td>
-              <td><code>"cov:methodCode": 1</code> OR <code>"cov:methodName": "GET"</code></td>
+              <td><code>"cov:methodName": "GET"</code></td>
             </tr>
             <tr>
               <td><code>writeproperty</code></td>
-              <td><code>"cov:methodCode": 3</code> OR <code>"cov:methodName": "PUT"</code></td>
+              <td><code>"cov:methodName": "PUT"</code></td>
             </tr>
             <tr>
               <td><code>observeproperty</code></td>
-              <td> <code>"cov:methodCode": 1</code> OR <code>"cov:methodName": "GET"</code> with <code>[ { "cov:optionName": "Observe", "cov:optionValue": 0 }]</code></td>
+              <td><code>"subprotocol":"cov:observe"</code></td>
```

Comment:
Since Observe can also be used with the FETCH method, it might be good to specify `"cov:methodName": "GET"` in addition to `"subprotocol":"cov:observe"`.

Reply:
Done with https://github.com/w3c/wot-binding-templates/pull/82/commits/8d36578677e1a4fac6b81c198bae6615cb02b4af